r2/r2/lib/system_messages.py

```diff
@@ -130,10 +145,70 @@ def send_ban_message(subreddit, mod, user, note=None, days=None, new=True):
 
     message += "\n\n" + ("you can contact the moderators regarding your ban "
         "by replying to this message. **warning**: using other accounts to "
-        "circumvent a subreddit ban is considered a violation of reddit's "
+        "circumvent a %(ban_type)s ban is considered a violation of reddit's "
         "[site rules](/rules) and can result in being banned from reddit "
-        "entirely.")
+        "entirely." % {"ban_type": ban_type})
 
     item, inbox_rel = Message._new(mod, user, subject, message, request.ip,
         sr=subreddit, from_sr=True)
     queries.new_message(item, inbox_rel, update_modmail=False)
+
+
+def send_mod_permission_message(subreddit, changer, mod, oldperms,
```

Comment:
You can't ban / mute a mod; they have to be demodded first.

Admins can easily silently adjust permissions via the CLI interpreter, but they rarely get involved in subs anyway unless something bad happens, and when that does happen, they deserve a notification i'd assume.